x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/alerts_cell_actions.cy.ts

```diff
@@ -127,49 +139,13 @@ describe.skip('Alerts cell actions', { tags: ['@ess', '@serverless'] }, () => {
     });
```

Comment:
looking at the implementation of the `openAddFilterPopover` method line 132, you can remove `cy.get(GLOBAL_SEARCH_BAR_ADD_FILTER).should('be.visible');` as we don't really need to check for visibility to be able to click on it

Reply:
also in the implementation of the `fillAddFilterForm` method line 134 you can delete the first 2 lines as they don't add any value:

```
cy.get(ADD_FILTER_FORM_FIELD_INPUT).should('exist');
cy.get(ADD_FILTER_FORM_FIELD_INPUT).should('be.visible');
```